go.mod

```diff
@@ -81,3 +81,5 @@ require (
 	google.golang.org/genproto v0.0.0-20230530153820-e85fd2cbaebc // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230530153820-e85fd2cbaebc // indirect
 )
+
+replace github.com/heroiclabs/nakama-common => ../nakama-common
```

Comment:
hm, I assume it's a leftover from local development?

Reply:
Yep, it's intentional as we'll wait for the corresponding `nakama-common` changes to be merged and then point `go.mod` to that commit and rebase.